pypeit/core/skysub.py

```diff
@@ -119,6 +119,11 @@ def global_skysub(image, ivar, tilts, thismask, slit_left, slit_righ, inmask=Non
 
     # Sky pixels for fitting
     inmask_in = thismask & (ivar > 0.0) & inmask & np.logical_not(edgmask)
+    if not np.any(inmask_in):
```

Comment:
Can you please make the following changes here: 1) rename inmask_in to gpm everywhere in global_skysub to be more clear. This is a change we are trying to propagate everywhere. 2) print out a more informative warning message like "Input gpm pixel mask is eveyrwhere False for this slit.  There is likely a problem with this slit."

Reply:
done.  am re-running DEIMOS tests just to be sure..